modules/raft/src/integrationTest/java/org/apache/ignite/raft/server/ItJraftCounterServerTest.java

```diff
@@ -270,21 +286,25 @@ private void startCluster() throws Exception {
             startServer(i, raftServer -> {
                 raftServer.startRaftGroup(COUNTER_GROUP_0, listenerFactory.get(), INITIAL_CONF, defaults());
                 raftServer.startRaftGroup(COUNTER_GROUP_1, listenerFactory.get(), INITIAL_CONF, defaults());
-            }, opts -> {});
+            }, opts -> {
```

Comment:
Old code looked better. You probably used auto-formatting for the entire file, right?